pkg/cmd/roachtest/monitor.go

```diff
@@ -46,7 +102,13 @@ type monitorImpl struct {
 	monitorGroup *errgroup.Group // monitor goroutine
 	monitorOnce  sync.Once       // guarantees monitor goroutine is only started once
 
-	expDeaths int32 // atomically
+	// expExactProcessDeath if true indicates that the monitor should expect exact process
```

Comment:
Nit: "...exact process deaths..." -> "...that a specified process, as denoted by the triple in expProcessHealth.get, is dead...".

Reply:
Done.